plaso/storage/file_interface.py

```diff
@@ -677,6 +713,17 @@ def GetEventDataByIdentifier(self, identifier):
     """
     return self._storage_file.GetEventDataByIdentifier(identifier)
 
+  def GetEventDataStreamByIdentifier(self, identifier):
+    """Retrieves specific event data stream.
```

Comment:
Change to "Retrieves a specific event data stream". Please change this for all other implementations of this method as well.

Reply:
ack, fixed 2 instances of the missing `a`